### ISSUE TITLE
Improve maintainability of library

### ISSUE THREAD

**Issue by StefSchenkelaars** [livingsocial/swagger_yard#77]
Currently it takes quite long for pull requests to get reviewed and merged. As discussed https://github.com/livingsocial/swagger_yard/pull/75#issuecomment-975667047, it would be nice if we could improve the community around this gem.

Steps we can take:
### Add more contributors to GitHub 
@nicksieger said he will try to track someone with admin rights.

### Simplify releasing to rubygems
Maybe we can automate the releasing to rubygems in some way. I know there are tools to automatically bump versions once PRs are merged.
At our company we use [this GitHub actions workflow](https://github.com/Drieam/rspec-github/blob/master/.github/workflows/publish.yml) to build and publish the gem to rubygems based on a GitHub release / tag.

**Comment by nicksieger** [comment]
One thought, should we create a new Github org for the projects and fork them there?

**Comment by StefSchenkelaars** [comment]
Well, if we can prevent another fork that would be nice. But then indeed admin permissions on the repo would be nice.

Maybe @chrismo can help out? He is the only public member of the organisation.

**Comment by nicksieger** [comment]
Yeah I asked Chris, and he can probably help out. Though, since the livingsocial org/company is sort of defunct, I think it might make sense to create a new org and then check with Github support to see if they can make the new forks the official ones.

**Comment by StefSchenkelaars** [comment]
Hmm, I wanted to wait a bit to see if we could get admin access but it doesn't feel like it. So I've been looking at the [forks network](https://github.com/livingsocial/swagger_yard/network) of this repo. It seems that [netlify](https://github.com/netlify/swagger_yard) made a fork with some contributions but I don't know if they accept external contributors. 

Maybe @kschlarman knows if it is possible to make the netlify fork the official fork? If we could become contributors, Nick can arrange the gem pushing rights.

If that's not the case we could use any of our personal forks or create a dedicated organisation. What do you think?
